srv/salt/_modules/wait.py

```diff
@@ -168,6 +198,13 @@ def until_mds(**kwargs):
     fscheck.wait_for_healthy_mds()
 
 
+def until_all_in(**kwargs):
```

Comment:
Add `osds` to the func name (e.g. `until_all_osds_in`) ?

Reply:
Renamed